flags/src/main/java/com/yahoo/vespa/flags/PermanentFlags.java

```diff
@@ -198,6 +198,14 @@
             APPLICATION_ID, HOSTNAME
     );
 
+    public static final UnboundListFlag<String> ENVIRONMENT_VARIABLES = defineListFlag(
+            "environment-variables", List.of(), String.class,
+            "A list of environment variables set for all services. " +
+                    "Each item should be on the form <ENV_VAR>=<VALUE>",
+            "Takes effect on service restart",
+            ZONE_ID, APPLICATION_ID
```

Comment:
Including cluster type/name as dimension would be useful, although not sure how we would wire it in through the `ModelContext`

Reply:
Future extension ...